### PR TITLE
docs: update crates

### LINKED ISSUE #43: Add velato/vello_svg to linebender crates
title.

### PR DESCRIPTION
**Pull request by simbleau**
Adds `vello_svg`, `velato`, and `parley`. Closes #43.

Note: Remove branch after merge.

### CODE REVIEW

**Comment by xStrom** [comment]
Parley is part of the Xilem stack.

**Comment by simbleau** [comment]
Should be good now. @xStrom

**Comment by dfrg** [comment]
Seems like we should also add fontique, a font enumeration and fallback library used by parley.

**Comment by simbleau** [comment]
Added

File: content/_index.md [hunk 1]
```diff
@@ -39,9 +39,10 @@ These crates are under active development.
 
    This crate provides a set of shared types for concepts that are important for drawing/stroking paths, but excluding the path geometry itself (which can be found in [`kurbo`][kurbo]). It includes types for brush styles (including gradient) and color.
 
-The following crates are not part of the `xilem`/`vello` stack, but are under active development.
-
- - [`velato`][velato] - A crate that renders Lottie animations to `vello`.
+ - [`parley`][parley] - A library for rich text layouts. It is backed by [`swash`][swash].
+ - [`fontique`][fontique] - A library for font enumeration and fallback library. Used by `parley`.
```
Comment (dfrg) [inline content/_index.md:43]: “library” is duplicated here.

**Review by dfrg — APPROVED** [review]
Drop the last “library” in the fontique description. Otherwise LGTM.

File: content/_index.md (hunk 1 above)
Comment (simbleau) [reply content/_index.md]: fixed. Thanks!

**Pull request merged**